src/styles/blocks/catalog.scss

```
@@ -0,0 +1,18 @@
+.catalog_container {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0 auto;
+  width: 1024px;
+  padding: 50px 40px;
+  align-items: center;
+  justify-content: flex-start;
+
+}
+
+.catalog--card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  margin-bottom: 46px;
+  margin-right: 48px;
+}
```

It seems like `.catalog--card` is being used as a modifier, but it should actually be an element of the catalog. The only styles that should be applied to the catalog__card are `margin-bottom: 46px` and `margin-right: 48px`. Styles such as `width` and `border` are internal card styles and should be placed in `.card`.

```suggestion
.catalog {
  &_container {
  display: flex;
  flex-wrap: wrap;
  margin: 0 auto;
  width: 1024px;
  padding: 50px 40px;
  align-items: center;
  justify-content: flex-start;

}

&__card {
  margin-bottom: 46px;
  margin-right: 48px;
  
  &:nth-child(4n) {
    margin-right: 0;
  }
}
}
```